src/client/SplitwiseClient.ts

```diff
@@ -1,6 +1,7 @@
 import Axios, { AxiosInstance } from "axios";
 import { appConfig } from "../config/AppConfig";
 import { ExpensesResponse } from "../types/Splitwise";
+import "axios-debug-log";
```

Comment:
I haven't checked but would be useful if we could reuse winston for logging with the http interceptor somehow :)
also for the sake of consistence logging format

Reply:
I think it'll be easier to just take this out for now - this was more for my personal poking around to see how this worked.